lldb/test/API/lang/swift/variables/generic_struct_debug_info/generic_apply/main.swift

```diff
@@ -14,7 +14,7 @@ func apply<Type>(_ T : Type, fn: (Type) -> Type) -> Type { return fn(T) }
 public func f<Type>(_ value : Type)
 {
   apply(value) { arg in
-    return arg //% self.expect('po arg', substrs=['3735928559'])
+    return arg //% self.expect('expr -o -d run TestSwiftGenericStructDebugInfoGenericApply.py -- arg', substrs=['3735928559'])
```

Comment:
Not sure if I understand this. I would say this is a typo but apparently it's passing?

```
PASS: lldb-api :: lang/swift/variables/generic_struct_debug_info/generic_apply/TestSwiftGenericStructDebugInfoGenericApply.py (222 of 345)
```

Reply:
It actually seems `expr` just ignores this stuff.

Reply:
Wow.

Reply:
I think this is an artifact of how arguments are parsed (and how I copied &pasted)